client/src/app.js

```diff
@@ -51,7 +51,13 @@ export default function main({ DOM, HTTP, route, storage, scanner: scan$, search
     , last_txids: parseHashes(loc.query.txids)
     , est_chain_seen_count: +loc.query.c || 0
     }))
-  , goAssetList$ = !process.env.IS_ELEMENTS || !process.env.ASSET_MAP_URL ? O.empty() : route('/assets')
+  //, goAssetList$ = !process.env.IS_ELEMENTS || !process.env.ASSET_MAP_URL ? O.empty() : route('/assets')
+  , goAssetList$ = !process.env.IS_ELEMENTS ? O.empty() : route('/assets/registry').map(loc => ({ 
```

Comment:
Is it intended to be `/assets/registry` and not `/assets`?

Reply:
What do you mean?

Reply:
The current asset list on the live website is at `https://blockstream.info/liquid/assets`, not `https://blockstream.info/liquid/assets/registry`. Did you mean to change it?

(the API endpoint does include `/registry`, but the frontend URL doesn't have to)

Reply:
I see... changes have been made